src/components/CarouselItem.tsx

```diff
@@ -111,7 +111,7 @@ export const CarouselItem = ({ animation, next, prev, swipe, state, index, maxIn
     let animate = 'center';
     if (index === active)
         animate = 'center';
-    else if (index === prevActive)
+    else if (maxIndex>1 && index === prevActive)
```

Comment:
This solution created a jitter effect where the current card would disappear and then would be replaced by the next card with a slide effect.. i have fixed that... 


```suggestion
    else if (index === prevActive)
```